code/datums/keybinding/mob.dm

```diff
@@ -1,8 +1,104 @@
 /datum/keybinding/mob
 		category = CATEGORY_HUMAN
 		weight = WEIGHT_MOB
+		
+		
+/datum/keybinding/mob/move_north
+	key = "W"
+	name = "move_north"
+	full_name = "Move North"
+	description = ""
+	keybind_signal = COMSIG_KB_MOB_MOVENORTH_DOWN
+
+/datum/keybinding/mob/move_north/down(client/user)
+	. = ..()
+	if(.)
+		return
+	if(!user.mob) return
```

Comment:
Properly newline indent the returns for consistency with the rest of the codebase

Reply:
On line 17? The rest of the file has the same indentation as 14-17, should I change those as well?